scripts/mail-alarm

```diff
@@ -388,6 +505,12 @@ class XapiMessage:
         except:
             self.body      = ""
         self.pool_name = get_pool_name()
+        try:
+            with open('/etc/xapi.d/mail-languages/' + mail_language + '.json', 'r') as fileh:
+                self.ml = json.loads(fileh.read())
+        except:
```

Comment:
Catch the error and add it to the message. It could fail for many different reasons: file does not exists, file unreadable, encoding/decoding error, invalid json. We need to know why it is failing explicitly